cloud/scope/suite_test.go

```diff
@@ -16,51 +16,79 @@ limitations under the License.
 package scope
 
 import (
-	"path/filepath"
+	"fmt"
+	"path"
 	"testing"
 
-	"github.com/onsi/ginkgo/reporters"
+	"github.com/golang/mock/gomock"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/client-go/rest"
-	capiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
-	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	ctrl "sigs.k8s.io/controller-runtime"
 
 	infrav1beta1 "sigs.k8s.io/cluster-api-provider-ibmcloud/api/v1beta1"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/pkg/cloud/services/vpc/mock"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/test/helpers"
+	capiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
this import should be in the above section with other imports

Reply:
Fixed.